src/Graphula/Node.hs

```diff
@@ -54,11 +56,9 @@ import UnliftIO.Exception (Exception, throwIO)
 -- 'NodeOptions' can be created and combined with the Monoidal operations '(<>)'
 -- and 'mempty'.
 --
--- @
--- a1 <- node @A () mempty
--- a2 <- node @A () $ edit $ \a -> a { someField = True }
--- a3 <- node @A () $ ensure $ (== True) . someField
--- @
+-- > a1 <- node @A () mempty
+-- > a2 <- node @A () $ edit $ \a -> a { someField = True }
+-- > a3 <- node @A () $ ensure $ (== True) . someField
```

Comment:
I'm having a lot of trouble getting the `@`/`@` style code blocks (which support `'linking'` identifiers) to work reliably with `@TypeApplications` present. There doesn't seem to be a pattern to when they break or are parsed correctly, vanish, or result in breaking the rest of the line. So I'm resorting to `>`-style in a lot of places.